src/NetMQ/NetMQRuntime.cs

```diff
@@ -38,7 +36,14 @@ public NetMQRuntime()
         /// </summary>
         public static NetMQRuntime Current
         {
-            get { return s_current.Value; }
+            get
+            {
+                var result = s_current.Value;
+                if (result == null)
+                    throw new InvalidOperationException("NetMQRuntime.Current is not available on this thread. Ensure that a NetMQRuntime has been created.");
+                else
+                    return result;
+            }
```

Comment:
Can be a little more succinct thanks to throw expressions, added to C# not so long ago.

```suggestion
            get => s_current.Value ?? throw new InvalidOperationException("NetMQRuntime.Current is not available on this thread. Ensure that a NetMQRuntime has been created.");
```